Tools/LambdaTestTool/src/Amazon.Lambda.TestTool/Runtime/LambdaExecutor.cs

```diff
@@ -81,6 +77,33 @@ public async Task<ExecutionResponse> ExecuteAsync(ExecutionRequest request)
             return response;
         }
 
+        public ILambdaContext GetLambdaContextForRequest(string requestLambdaContext, ILambdaLogger defaultLogger)
+        {
+            if (string.IsNullOrWhiteSpace(requestLambdaContext))
+            {
+                return new LocalLambdaContext()
+                {
+                    Logger = defaultLogger
+                };
+            }
+
+            var json = JsonConvert.DeserializeObject<LocalLambdaContext>(requestLambdaContext); 
+
+            return new LocalLambdaContext
```

Comment:
I'm wondering why not just return `json` here since it will be a `LocalLambdaContext`?